kairos-prover/rust-std-Cargo.lock

```diff
@@ -0,0 +1,6636 @@
+# This file is automatically @generated by Cargo.
```

Comment:
**Question:** where is TOML file used to generate this lockfile?

Reply:
There is none because rust-std only ships with a lockfile: I don't know how you obtain your rust compiler but there should be a directory `lib/rustlib/src/rust` and this is where this fiel is copied from